fundingmanager.go

```diff
@@ -3148,6 +3149,20 @@ func (f *fundingManager) handleInitFundingMsg(msg *initFundingMsg) {
 		return
 	}
 
+	// If the caller specified to spend all funds, then we define the
+	// maximum allowed funding amount depending on if both local and remote
+	// peer supports wumbo channels.
+	var fundUpToMaxAmt btcutil.Amount
+	if msg.fundMax {
+		fundUpToMaxAmt = MaxFundingAmount
+		remoteWumbo := msg.peer.RemoteFeatures().HasFeature(
+			lnwire.WumboChannelsOptional,
+		)
+		if !f.cfg.NoWumboChans && remoteWumbo {
```

Comment:
As far as I understand it the fundingmanager has the field `NoWumboChannels` inactive if Wumbo is enabled with `--protocol.wumbo-channels`:
https://github.com/lightningnetwork/lnd/blob/77549f1fb8f046fc565d42151a4ae3900391170e/server.go#L970

So there is no need to check what the local node is advertising as enabled features and it suffices to check if Wumbo is enabled in the fundingmanager and advertised as features with the remote node?